code_of_conduct.md

```diff
@@ -0,0 +1,37 @@
+# Code of Conduct
+
+The Women Who Code DC Mission has three main goals:
+1. That women software engineers feel confident and empowered to pursue their goals.
+1. That every woman codes
```

Comment:
the grammar on this is a bit funny - maybe "That every woman has the ability to code"